Missing babel-types dependency?

I'm building a [storybook](https://github.com/storybooks/storybook) based module and coming up against this error:

```
Module build failed: Error: Cannot find module 'babel-types'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/path/to/project/node_modules/babel-plugin-styled-components/lib/visitors/minify.js:7:19)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

It looks like `/src/visitors/minifiy.js` does indeed try to [import `babel-types`](https://github.com/styled-components/babel-plugin-styled-components/blob/master/src/visitors/minify.js#L1) but the dependency is missing from this module's `package.json` file.  Any chance of adding it?

--- a/package.json
+++ b/package.json
@@ -33,6 +33,7 @@
     "displayName"
   ],
   "dependencies": {
+    "babel-types": "^6.26.0",
     "stylis": "^3.2.1"
   }
 }

